pinot-controller/src/test/java/org/apache/pinot/controller/api/ControllerFilePathProviderTest.java

```diff
@@ -0,0 +1,116 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pinot.controller.api;
+
+import java.io.File;
+import java.net.URI;
+import org.apache.commons.configuration.BaseConfiguration;
+import org.apache.commons.io.FileUtils;
+import org.apache.pinot.controller.ControllerConf;
+import org.apache.pinot.controller.api.resources.ControllerFilePathProvider;
+import org.apache.pinot.filesystem.PinotFSFactory;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertNotNull;
+import static org.testng.Assert.assertTrue;
+
+
+public class ControllerFilePathProviderTest {
+  private static final String HOST = "localhost";
+  private static final String PORT = "12345";
+  private static final File DATA_DIR = new File(FileUtils.getTempDirectory(), "ControllerFilePathProviderTest");
```

Comment:
"this.getClass().getName()"

Reply:
+1

Reply:
This is static, so equivalent to `ControllerFilePathProviderTest.class.getSimpleName()` which IMO is not as readable. Most tests are directly using the string.